src/OpenLoco/Core/Optional.hpp

```diff
@@ -21,5 +15,6 @@ namespace std
     using std::experimental::optional;
 }
 #endif
-
-#undef NORMAL_OPTIONAL // Not needed any more, don't make it public.
+#else
+#error "<optional> header not found"
```

Comment:
This isn't true though. It's if has include is defined.

Reply:
```suggestion
#error "<optional> header could not be located"
```

Reply:
That would be just as wrong. This branch will only be hit by compilers that don't implement has include. It should state error compiler does not implement has include

Reply:
You're right. I think we should still say it's trying to locate the <optional> header, though. What about the following:
```suggestion
#error "__has_include operator is unavailable; <optional> header could not be located"
```

Reply:
@duncanspumpkin Good to go?

Reply:
Sure